src/request/export/Export.css

```diff
@@ -3,3 +3,56 @@ button.go-to-file {
     display: block;
     margin: 4rem auto 0;
 }
+
+.page#download-key-file.show-download .hide-for-download {
+    display: none;
+}
+
+.page#download-key-file:not(.show-download) .download-key-file {
+    display: none;
+}
+
+.page#download-key-file .download-icon {
+    align-self: center;
+}
+
+.page#download-key-file .walletfile,
+.page#download-key-file .arrow-down {
+    display: block;
+    margin: auto;
+}
+
+.page#download-key-file .walletfile {
+    width: 9rem;
+    height: 11.75rem;
+    width: 100%;
+}
+
+.page#download-key-file .arrow-down {
+    width: 3rem;
+    height: 3rem;
+    opacity: 0.3;
+    margin-top: 1rem;
+}
+
+#privacy.page.key-active .hide-if-key-active {
+    display: none;
+}
+
+#privacy.page:not(.key-active) .page-footer button:not(.submit) {
```

Comment:
Like the selectors above, `#privacy.page` can be switched around to `.page#privacy`.